timestamp_test.go

```diff
@@ -329,6 +329,20 @@ func ExampleCreateRequest_customHashingAlgorithm() {
 	}
 }
```

Comment:
Can you add a test for successful parsing?

Reply:
Yep, just added one.